cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -1342,6 +1360,31 @@ TEST_F(TestUInt32ParquetIO, Parquet_1_0_Compatibility) {
 
 using TestStringParquetIO = TestParquetIO<::arrow::StringType>;
 
+TEST_F(TestStringParquetIO, Basics) {
```

Comment:
Could we use some meaningful name to replace `Basics` here?

Reply:
Thanks for adding this test!

Reply:
So.. Based on your last comment, should I remove this test or rename it?

Reply:
Sorry for the confusion! Though simple string type won't cause any issue even before this change, it would be good to add this case to make sure large variant can read it. So let's not remove it. :)